engineering/workflow/default-workflow.md

```diff
@@ -60,29 +60,148 @@ Last day of Iteration
 
 Attendees of the Refinement meeting should include representatives from the following teams:
 - Engineering: for technical decision-making
 - Product: for product decision-making
-- Partnerships and Community Success: for community/grant decision-making.
+
+Product Managers at the meeting should ensure to come prepared to represent Business Development interests.
 
 (meetings:sprint-planning)=
-### 2. Iteration Planning
 
-The team conducts a synchronous iteration planning meeting for 60 minutes at the beginning of each iteration.
+### 2. Iteration Planning Guide
+
+#### 🗓️ Between Refinement and Planning
+
+A Product Manager will:
+
+- **Add deadlines to issues that need them**  
+  A Product Manager will ensure that any issue with a contractual obligation or expected delivery has a clear due date.
+
+- **Mark contract deliverables with dates**  
+  Use labels to easily identify Contract Deliverables.
+
+- **Assign preferred sprint-end dates**  
+  If a card is intended for the upcoming sprint, let a Product Manager know and they will mark it with this sprint’s end date.
+
+- **Organize in 'Up Next'**  
+  A Product Manager manages and sorts cards in the ‘Up Next’ column for visibility.
+
+The Team will:
+
+- **Review open-source contributions**  
+  Include relevant contributions and initiatives in planning discussions.
+
+- **Advocate for priority cards**  
+  Team members should speak up for issues they’re most invested in or want to lead. Reach out to a Product Manager in the days leading up to a sprint to discuss the priority of the issues you wish to champion:
+  - First check that the issues you want to suggest are not already in the Up Next column and dated for the upcoming sprint. 
+  - Send a message to the PM at least one day prior to the planning session with a list of issues you feel need to be included in the upcoming sprint.
+  - Please include as much context as needed for the PM to understand why these issues need to be in the upcoming sprint
+  - The PM will take your suggestions under advisement and make a best effort to include them, but cannot guarantee that to be the case as other priorities and capacity will need to be considered. 
+
+#### 🗺️ Roadmap & Initiative Review
+
+The Product Manager will:
+
+- **Review roadmap tab for issues with dates**  
+  - Pull in overdue items.
+  - Reassign unrealistic dates to achievable ones.
+  - Ensure no items lag behind without follow-up.
+ 
+#### ✅ Preparing for Planning
+
+The Team will:
+
+- **Ensure issues are current**  
+  - All team members should update and clean up any issues to which they are assigned prior to the planning session
+  - Ensure the use of the Standard Definition of Done available in the General Issue template where possible.
+
+The Engineering Manager will:
+
+- **Report on last sprint’s progress**  
+  Add totals for:
+  - ✅ Done  
+  - 📂 Open  
+  - ❌ Unplanned  
+  to the previous sprint’s **Status Update**.
+
```

Comment:
I would like us to add here that 'The product manager who did these things will announce that it's done in slack channel X' or similar, as a marker.